torch/_inductor/codegen/cpp_prefix.h

```diff
@@ -57,6 +58,20 @@ template <typename T>
 struct IsVecType<at::vec::Vectorized<T>>: std::true_type {};
```

Comment:
not related to this PR but do we also need to check `at::vec::VectorizedN` too?

Reply:
Will also consider `at::vec::VectorizedN` in the future PR of using full bits width for /bf16/fp16.

Reply:
`torch.var` reduces in float precision so I don't think `VectorizedN` ever comes up.

Reply:
It will after we support float16/bfloat16 with vector factor of 32. Then, we would have VectorizedN<float, 2> to match that.